spec/mailers/admin_mailer_spec.rb

```diff
@@ -10,15 +10,33 @@
       let(:comment) { create(:comment, :on_admin_post) }
 
       context "and the comment's contents contain an image" do
-        let(:image_tag) { "<img src=\"an_image.png\" />" }
+        let(:image_url) { "an_image.png" }
+        let(:image_tag) { "<img src=\"#{image_url}\" />" }
 
         before do
           comment.comment_content += image_tag
           comment.save!
         end
 
-        it "strips the image from the email message" do
-          expect(email).not_to have_html_part_content(image_tag)
+        context "when image safety mode is enabled for admin post comments" do
```

Comment:
This context should check text content, like its counterpart for `#edited_comment_notification` below.